cli/cmd/tap.go

```diff
@@ -281,10 +281,10 @@ func renderTapEvent(event *common.TapEvent) string {
 			ev.ResponseInit.GetSinceRequestInit().Nanos/1000,
```

Comment:
I rolled back to the commit where the initial tap panic was reported in #47, and it looks like it's a result of this `.Nanos` call. It should be:

```
			ev.ResponseInit.GetSinceRequestInit().GetNanos()/1000,
```

I think you should also update all of the `.Nanos` calls as part of this branch.

Reply:
Looks like you missed this `.Nanos` call. Otherwise LGTM.

Reply:
heh, sed gone wrong